src/deltaE/deltaEJz.js

```diff
@@ -39,7 +39,7 @@ Color.prototype.deltaEJz = function (sample) {
 	}
 
 	let Δh = Hz1 - Hz2;
-	let ΔH = 2 * Math.sqrt(Cz1 * Cz2) * Math.sin(Δh * Math.PI / 180);
+	let ΔH = 2 * Math.sqrt(Cz1 * Cz2) * Math.sin((Δh / 2) * (Math.PI / 180));
```

Comment:
The argument to Math.sin is in radians.

The original code converts degrees to radians (but fails to divide by two)
The new code passes the value in degrees and then multiplies the result by Π/180 !

We actually want 

```js
Math.sin((Δh / 2) * (Math.PI / 180));

Reply:
Cool, I'll take a look today.

Reply:
@svgeesus What I provided in this pull does not pass the value by degrees. It is actually identical to what you are suggesting. I suspect maybe a trick of the eye when looking at the parenthesis in the delta?

Reply:
Yes, you are right, my bad